pkg/server/server.go

```diff
@@ -425,6 +426,17 @@ func WithDispatchThrottlingCheckResolverMaxThreshold(maxThreshold uint32) OpenFG
 	}
 }
 
+// WithRequestContextPropagation determines whether the request context is propagated to the datastore.
+// When enabled, the datastore receives cancellation signals when an API request is cancelled.
+// When disabled, datastore operations continue even if the original request context is cancelled.
+// Disabling context propagation is normally desirable to avoid unnecessary database connection churn.
+// If not specified, the default value is false (separate storage and request contexts).
```

Comment:
the current default is true, please keep that as is

Reply:
See comment above.

Reply:
ah sorry, my bad - the default right now is that context is _not_ propagated 😆